### PR TITLE
[SW] Added dependents additional info explainer

### PR DESCRIPTION
**Pull request by josephrlee**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

Added an additional info explainer component explaining what a dependent is to the dependents pages.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#63175

## Testing done

Tested locally

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Dependents count  | staging va gov_manage-va-debt_request-debt-help-form-5655_dependents-count | localhost_3001_manage-va-debt_request-debt-help-form-5655_dependents-count (1) |
| Dependents ages | staging va gov_manage-va-debt_request-debt-help-form-5655_dependents-count (1) | localhost_3001_manage-va-debt_request-debt-help-form-5655_dependent-ages |

## What areas of the site does it impact?

Affects the FSR form.

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/financial-status-report/components/household/DependentAges.jsx [hunk 1]
```diff
@@ -169,7 +169,7 @@ const DependentAges = ({ goToPath, isReviewMode = false }) => {
       ? 'form-review-panel-page-header vads-u-font-size--h5'
       : 'schemablock-title vads-u-margin-top--5';
   const text =
-    isReviewMode && !isEditing ? 'Review Dependants ages' : 'Dependents ages';
+    isReviewMode && !isEditing ? 'Dependants ages' : 'Dependents ages';
```
Comment (amponce) [inline src/applications/financial-status-report/components/household/DependentAges.jsx:172]: So I think if we want the title to just be Dependent ages we can update this:

` <HeaderTag className={className}>{text}</HeaderTag>`
 to         
 `<HeaderTag className={className}>Dependent ages</HeaderTag>`
 
 and remove this ```const text =
    isReviewMode && !isEditing ? 'Dependants ages' : 'Dependents ages';```

File: src/applications/financial-status-report/pages/income/dependents/index.js [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+    <p>Here’s who we consider dependents:</p>
+    <ul>
+      <li>Your spouse</li>
+      <li>Unmarried children who are under 18 years old</li>
+      <li>Adult children who were disabled before age 18</li>
+      <li>Children ages 18 to 23 who attend school full time</li>
+    </ul>
+  </va-additional-info>
+);
+
```
Comment (amponce) [inline src/applications/financial-status-report/pages/income/dependents/index.js:17]: In the components/household directory, I recommend creating a new component named DependentExplainer. You can add the following code snippet, just as you've done, but be sure to include the `export` keyword:

```
import React from 'react';

export const DependentsExplainer = (
  <va-additional-info
    trigger="Who qualifies as a dependent?"
    class="vads-u-margin-top--2"
  >
    <p>Here’s who we consider dependents:</p>
    <ul>
      <li>Your spouse</li>
      <li>Unmarried children who are under 18 years old</li>
      <li>Adult children who were disabled before age 18</li>
      <li>Children ages 18 to 23 who attend school full time</li>
    </ul>
  </va-additional-info>
);
```

Next, you can import the DependentsExplainer component into pages/income/dependents:
```import { DependentsExplainer } from '../components/household/DependentsExplainer';```
And update the 'ui:description' property to use DependentsExplainer:
```
'ui:description': DependentsExplainer,
```
Now, you can make a change in the DependentAges component. 
Replace the existing code block with the concise use of the DependentsExplainer component:
```
{dependentAgeInputs}
{!isReviewMode ? (
  <>
    <DependentsExplainer />
  </>
) : null}
```
Lastly, don't forget to import the DependentsExplainer component at the beginning of the file:
```
import { DependentsExplainer } from './DependentsExplainer';
```

**Review by amponce — COMMENTED** [review]
Let me know if you want to pair on this and we can knock it out together

**Review by amponce — APPROVED** [review]
Nice work! 🚀

**Comment by josephrlee** [comment]
Closing this PR due to major CI issues. Starting new PR #25204 from latest to get changes in.

cc: @amponce

**Pull request closed without merging**